CONTRIBUTING.md

```diff
@@ -0,0 +1,20 @@
+Issues
+------
+
+If you've found a bug or have a great idea for a new feature that will help all users of impress.js, let us know by [adding your suggestion]
+(http://github.com/bartaz/impress.js/issues/new) to the [issues list](https://github.com/bartaz/impress.js/issues).
+
+Guidelines:
+
+* If reporting a bug, please add a [simplified example](http://sscce.org/).
```

Comment:
I recommend to point websites like [Pastebin](http://pastebin.com/) or [JsFiddle](https://jsfiddle.net/) to be sure users know about them.